ansible_runner/streaming.py

```diff
@@ -244,8 +244,11 @@ def run(self):
             try:
                 line = self._input.readline()
                 data = json.loads(line)
-            except (json.decoder.JSONDecodeError, IOError):
-                self.status_callback({'status': 'error', 'job_explanation': 'Failed to JSON parse a line from worker stream.'})
+            except (json.decoder.JSONDecodeError, IOError) as exc:
```

Comment:
@AlanCoding Thanks for the patch. Could you please add a test for this change?

Reply:
Sorry for the delay. I agree.

I just pushed a change where I modified some existing tests. There was effectively already coverage, but I added more detail to the assertions.